README.md

```diff
@@ -1,79 +1,81 @@
 
 ### Colony setup
 
-#### Link you AWS account and S3 bucket to Colony
+#### Link your AWS account and S3 bucket to Colony
 
-Go to **Settings** and link your aws account on **Cloud Account** page and set your s3 bucket as an artifacts repository
-on **Integrations** page
+Go to **Settings** and link your aws account on **Cloud Account** page. Then, set your s3 bucket as an artifacts repository
+on **Integrations** page.
 
 #### Connect blueprint repo to your Colony space
 
-In your space go to **Settings -> Blueprint Repositories** and connect this repo as a blueprint repository for Sandbox Environment.
-If you want to do some experiments with code and make some changes fork it and connect your own repository
+In your space, go to **Settings -> Blueprint Repositories** and connect this repo as a blueprint repository for Sandbox Environment.
+If you want to experiment with the code and make some changes, fork it and connect your own repository.
 
 #### Activate blueprints
 
-Go to Blueprints and publish both Jenkins and Promotions Manager blueprints in "Space Blueprints" list
+Go to **Blueprints** and publish both Jenkins and Promotions Manager blueprints in the **Space Blueprints** list.
 
-Now you can start working with the Jenkins blueprint
+You may now start working with the Jenkins blueprint.
 
 ### Jenkins Blueprint Usage
 
-Now you are ready to launch a sandbox with Jenkins on Sandbox Environment page
+Now you are ready to launch a sandbox with Jenkins on **Sandbox Environment** page.
 
 Jenkins Blueprint has the following list of input parameters:
 
-*AWS_INSTANCE_TYPE* - specify the type of aws instance according to required size (default is `m5.large`)
+* `AWS_INSTANCE_TYPE` - specify the type of AWS instance according to required size (default is `m5.large`).
 
-*JENKINS_ADMIN_PASS* - set the password which you will use when logging in to Jenkins with `admin` user. Default is "admin"
+* `JENKINS_ADMIN_PASS` - set the password you will use when logging in to Jenkins with the admin user. Default is `admin`.
 
-*CS_COLONY_TOKEN* - token which Jenkins will use to communicate with Colony. You can generate it on **Settings -> Integrations** page
+* `CS_COLONY_TOKEN` - token that Jenkins will use to communicate with Colony. You can generate it on **Settings -> Integrations** page.
 
-*IAM_ROLE* - IAM role instance profile name you've created at the beginning
+* `IAM_ROLE` - IAM role instance profile name you've created at the beginning.
 
-*SPACE_NAME* - the name of the space which Jenkins will use to run sandboxes from. For our example just specify the name of your current space. **Note**: In future you might want to create your personal blueprints and configure Jenkins pipelines to use them. These blueprints will probably live in other spaces.
+* `SPACE_NAME` - the name of the space which Jenkins will use to run sandboxes from. For our example, just specify the name of your current space.
```

Comment:
the name of the space in which Jenkins will run sandboxes. For our purposes, just specify the name of your current space.